### ISSUE TITLE
[Bug] docker-compose breaking

### ISSUE THREAD

**Issue by mkubdev** [SleekwareDB/sleekwaredb#33]
Hi,

`docker-compose up` throw the error :
```bash
WARNING: The BACKEND_PORT variable is not set. Defaulting to a blank string.
ERROR: The Compose file './docker-compose.yml' is invalid because:
services.sleekwaredb-nginx.ports contains an invalid type, it should be a number, or an object
```

**Comment by darkterminal** [comment]
How about your `.env` file? Is initialize right?

**Comment by mkubdev** [comment]
Sure i tried `docker-compose up` in root directory.

I fixed this 'issue' with the PR #35
